source/vistas/core/color.py

```diff
@@ -117,7 +123,7 @@ def hsva_list(self):
 
 
 def interpolate_color(range, min_color: Color, max_color: Color, t):
-    """ Linearly interpolate in the HSV color space """
+    """ Linearly interpolate in the HSV color space. Copies colors in RGB space to HSV. """
```

Comment:
The conversion feels like the more salient point. Copying is more an ancillary detail of the conversion mechanism.